rtd-requirements.txt

```diff
@@ -22,3 +22,4 @@ flask-restful>=0.3.3
 ordereddict>=1.1
 flask-restplus>=0.7.2
 cherrypy
+python-telegram-bot>=1.8
```

Comment:
I think this is too aggressive. Pushbullet and the like are optional plugins. It should be up to the user to install such dependencies as no core functionality depends on it.